src/app/pages/network/ipmi/ipmi.component.ts

```diff
@@ -132,6 +141,17 @@ export class IPMIComponent {
     if (window.localStorage.getItem('is_freenas') === 'false') {
       this.ws.call('failover.licensed').subscribe((is_ha) => {
         this.is_ha = is_ha;
+        if (this.is_ha) {
+          this.ws.call('failover.node').subscribe((node) => {
+            if (node === 'A') {
```

Comment:
This whole block can be replaced with the ternary operator.  That assumes we will only ever have two controllers, but the if/else if block also does that.
```
this.currentControllerLabel = (node === 'A') ? '1' : '2';
this.failoverControllerLabel = (node === 'A') ? '2' : '1';
```